src/MetaModels/ItemList.php

```diff
@@ -752,6 +770,8 @@ public function render($blnNoNativeParsing, $objCaller)
             ->dispatch(MetaModelsEvents::RENDER_ITEM_LIST, $event);
 
         $this->objTemplate->noItemsMsg = $this->getNoItemsCaption();
+        // Backwards compatibility with templates as of #1078
+        $this->objTemplate->details = $this->getDetailsCaption();
```

Comment:
For C4 I re-added it as `$this->objTemplate->details = $this->getCaptionText('details');` which saves us adding yet another protected method.
What do you think?

Reply:
+1
Sure, we don't need the `getDetailsCaption()` necessarily, the method was just present until I removed in #1078

Reply:
I know, so let's keep the removal but reintroduce the template value.

Reply:
Done.